src/test/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStreamTest.java

```diff
@@ -69,19 +69,19 @@ public void splitZipBeginsWithZipSplitSignature() throws IOException {
     public void testCreateSplittedFiles() throws IOException {
         final File testOutputFile = new File(dir, "testCreateSplittedFiles.zip");
         final int splitSize = 100 * 1024; /* 100KB */
-        final ZipSplitOutputStream zipSplitOutputStream = new ZipSplitOutputStream(testOutputFile, splitSize);
+        final ZipSplitOutputStream ZipSplitOutputStream = new ZipSplitOutputStream(testOutputFile, splitSize);
 
         final File fileToTest = getFile("COMPRESS-477/split_zip_created_by_zip/zip_to_compare_created_by_zip.zip");
         final InputStream inputStream = Files.newInputStream(fileToTest.toPath());
         final byte[] buffer = new byte[4096];
         int readLen;
 
         while ((readLen = inputStream.read(buffer)) > 0) {
-            zipSplitOutputStream.write(buffer, 0, readLen);
+            ZipSplitOutputStream.write(buffer, 0, readLen);
```

Comment:
-1 ditto

Reply:
My mistake, don't know how this happened or why I modified this test :/